plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1373,10 +1373,6 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
         return true
       }
 
-      if (detector.pointersCount > 1) {
```

Comment:
what would happen if we use 3 fingers? 😄 
I'd better go with imo better replace it with `detector.pointersCount > 2` condition

Reply:
👍 good point, validated that other map seem also block 3 finger gestures, will update.

Reply:
Do we really need to consider 3 fingers? It seems on my device if you are using 3 fingers, the system will catch it and take a screen shot

Reply:
> It seems on my device if you are using 3 fingers, the system will catch it and take a screenshot
I guess it's device specific, however tested on my phone, if we don't add this check, 3 fingers move will trigger the pan gesture, while with Google Maps, 3 finger pan is blocked. We should limit it to only allow 1 or 2 finger pans imo.